x/rollup/keeper/deposits.go

```diff
@@ -121,13 +121,15 @@ func (k *Keeper) processL1UserDepositTxs(
 			ctx.Logger().Error("Failed to get sender address", "evmAddress", from, "err", err)
 			return nil, types.WrapError(types.ErrInvalidL1Txs, "failed to get sender address: %v", err)
 		}
-		cosmAddr := utils.EvmToCosmosAddress(from)
+		mintAddr := utils.EvmToCosmosAddress(from)
```

Comment:
question to reviewers - do you think `senderAddr` or `fromAddr` is a better name here than `mintAddr`?